pkg/routeagent/controllers/route/route.go

```diff
@@ -419,36 +422,47 @@ func (r *Controller) processNextEndpoint() bool {
 			return fmt.Errorf("error retrieving submariner endpoint object %s: %v", name, err)
 		}
 
+		klog.V(log.TRACE).Infof("Processing endpoint %s : %s", name, endpoint.String())
+
 		if endpoint.Spec.ClusterID != r.clusterID {
-			klog.V(6).Infof("setting routes of endpoint object %s", name)
+			klog.Infof("Updating iptable rules for remote cluster's Endpoint %q with subnets %v",
+				endpoint.Name, endpoint.Spec.Subnets)
 			r.updateIptableRulesForInterclusterTraffic(endpoint.Spec.Subnets)
 			r.endpointWorkqueue.Forget(obj)
 			return nil
 		}
 
+		klog.Infof("Processing local Endpoint %s with IP %s, Host %s for this cluster",
+			endpoint.Name, endpoint.Spec.PrivateIP, endpoint.Spec.Hostname)
+
 		hostname, err := os.Hostname()
 		if err != nil {
 			klog.Fatalf("unable to determine hostname: %v", err)
 		}
 
-		klog.V(6).Infof("Local Cluster Gateway Node IP is %s", endpoint.Spec.PrivateIP)
-
 		// If the endpoint hostname matches with our hostname, it implies we are on gateway node
 		if endpoint.Spec.Hostname == hostname {
-			r.cleanRoutes()
+			klog.Infof("This route agent is running on the active gateway node")
+
 			r.gwVxLanMutex.Lock()
 			defer r.gwVxLanMutex.Unlock()
 
 			r.isGatewayNode = true
+			klog.Infof("Creating the vxlan interface: %s on the gateway node", VxLANIface)
 			err = r.createVxLANInterface(VxInterfaceGateway, nil)
 			if err != nil {
-				klog.Fatalf("Unable to create VxLAN interface on GatewayNode (%s): %v", hostname, err)
+				klog.Fatalf("Unable to create VxLAN interface on gateway node (%s): %v", hostname, err)
 			}
-			klog.V(6).Infof("not reconciling routes because we appear to be the gateway host")
+
+			r.cleanRoutes()
+
+			klog.V(log.DEBUG).Infof("not reconciling gateway routes because we appear to be the gateway host")
```

Comment:
Not sure this really adds any value since we've already reported we're on the gateway host above.

Reply:
Ack, let's remove it.

Reply:
Done.